### ISSUE TITLE
Include website address does not work anymore after updating alias settings

### ISSUE THREAD

**Issue by Heziode** [simple-login/app#961]
Please note that this is only for bug report. 

For help on your account, please reach out to us at hi[at]simplelogin.io. Please make sure to check out [our FAQ](https://simplelogin.io/faq/) that contains  frequently asked questions.

For feature request, you can use our [forum](https://github.com/simple-login/app/discussions/categories/feature-request). 

For self-hosted question/issue, please ask in [self-hosted forum](https://github.com/simple-login/app/discussions/categories/self-hosting-question)

## Prerequisites
- [X] I have searched open and closed issues to make sure that the bug has not yet been reported.

## Bug report

**Describe the bug**
When I do some modifications in **Aliases** section, the **Include website address** feature does not work anymore. Even if I disable > update > enable > update it.

**Expected behavior**

The feature **Include website address** should work (or be automatically disabled) when we edit **Aliases** parameters.

**Screenshots**

![CleanShot 2022-05-10 at 11 45 23@2x](https://user-images.githubusercontent.com/16735076/167600568-4c2bbbf3-2450-407e-bbf5-0d0ce8adc8a4.png)

<img width="963" alt="CleanShot 2022-05-10 at 11 47 01@2x" src="https://user-images.githubusercontent.com/16735076/167600643-6d25a230-c0e0-4811-a713-67badb3a1f06.png">

**Additional context**
My version of the backend is based on 67dad33b700d86c9364092f8efa32fe289fcc5e0 (like the current [app.simplelogin](https://app.simplelogin.io/git))

**Comment by nguyenkims** [comment]
I tried to reproduce the issue following these steps but can't, let me know if I'm missing something:
- select my custom domain as default domain for aliases
- disable the "Include website address in one-click alias creation on browser extension" setting
- click a new alias via the browser extension by right clicking on "Create random alias": the alias doesn't have the website name
- enable the "Include website address in one-click alias creation on browser extension" setting
- click a new alias via the browser extension by right clicking on "Create random alias": the alias has the website name

**Comment by Heziode** [comment]
The problem does not came from the Popup, but from the added button in forms, like in the following screenshot:

![CleanShot 2022-05-12 at 10 26 36@2x](https://user-images.githubusercontent.com/16735076/168026726-7fa51845-e901-424d-8783-cf24216807df.png)

If I click on button the following button, it generate an alias with the website address:

![CleanShot 2022-05-12 at 10 27 40@2x](https://user-images.githubusercontent.com/16735076/168027003-fdbda41d-30d6-4f06-b30b-c15170ac1eb6.png)

However, if I use the button in a form (first screenshot), it generate a complete random alias.

**Comment by nguyenkims** [comment]
Yes this second button indeed always generate a random alias. It's a bit confusing and we might remove it in the next version.

**Comment by Heziode** [comment]
So, for the button added in form, it should act like the `Create` button of the popup of the extension, isn’t?

Because this is not the case, since it does not generate an email based on the domain name (like explained in the first post)

**Comment by nguyenkims** [comment]
I'm not sure to totally understand. Currently there are 3 ways to create an alias via the browser extension:
- clicking on the icon injected in the email field: the new alias uses the website domain if the setting is enabled (it's enabled by default) and the default domain if set.
- right click: create the same alias as clicking on the icon
- click on "OR create a totally random alias": create a total random alias, not using the website domain or default domain. Will probably removed in the next version.

**Comment by Heziode** [comment]
> clicking on the icon injected in the email field: the new alias uses the website domain if the setting is enabled (it's enabled by default) and the default domain if set.

I mean this one.
The behavior is weird. When I test this button on [Facebook](https://facebook.com) it generates an alias based on the website domain, however, if I attempt to generate an alias on [just-eat](https://www.just-eat.fr), it generates a random alias.

**Comment by nguyenkims** [comment]
@Heziode I went to https://www.just-eat.fr/ and the proposed alias starts indeed with `just-eat`, I wonder if your setting has maybe changed in the meantime?

**Comment by Heziode** [comment]
@nguyenkims, 

Here is a short video of the issue:

https://user-images.githubusercontent.com/16735076/169049298-0fb852c0-5aec-4daf-80dc-eab211acb272.mp4

**Comment by nguyenkims** [comment]
@Heziode thanks. I wonder if this alias has been used before on that website? Because in this case clicking on SL icon will return this same alias.

**Comment by Heziode** [comment]
Nope, this alias as not been used before. Every time I click on the button, it generates a new random alias.

**Comment by nguyenkims** [comment]
@Heziode can you send a screencast of what happens on Facebook? I don't have any clue for now why the domain is included for facebook.com but not https://www.just-eat.fr/ on your account.

**Comment by Heziode** [comment]
In fact, it doesn't even work on Facebook:

https://user-images.githubusercontent.com/16735076/169282668-0e5cb51a-75ea-469b-8463-688d98b81455.mp4

**Comment by nguyenkims** [comment]
I wonder if this can be due to the browser extension cannot extract the domain name from the tab? Can you try on another browser?

**Comment by Heziode** [comment]
Exactly the same issue using a fresh new FireFox install.

**Comment by nguyenkims** [comment]
We have released the new version 2.6.3 on Firefox, I'd suggest to give this version a try. For now I don't have any idea about what can cause the issue you exprienced.

**Comment by Heziode** [comment]
Same issue with 2.6.3 on FireFox.

Is the order / priority of the generation of the local part is saved into the database? Maybe an issue on this side

**Comment by nguyenkims** [comment]
@Heziode every call is different (not cached) so I don't think it's the case. Can you please create a new account with SL and retry? Thanks for your help!

**Comment by Heziode** [comment]
I didn't forget, I'm busy, I do the test when I have a moment.
